File: Modul 2/Python/PRAK204-2310817220025-FirdaKhoirunisa.py
print("Input")
r,t = map(int, input().split())

phi = 22/7

volume = phi*pow(r, 2)*t
luas = 2*phi*pow(r, 2)+2*phi*r*t
keliling = 2*phi*r

print("\nOutput")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul 2/C/PRAK204-2310817220025-FirdaKhoirunisa.c
#include<stdio.h>
#include<math.h>

void main(){
    double r, t;

    printf("Input\n");
    printf("");
    scanf("%lf %lf", &r, &t);

    double phi = 3.14286;
    double volume = phi*pow(r, 2)*t;
    double luas = 2*phi*pow(r, 2)+2*phi*r*t;
    double keliling = 2*phi*r;

    printf("\nOutput\n");
    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf", keliling);
}